cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -188,6 +192,26 @@ def enable_disable_migration_ui(appliance):
     appliance.disable_migration_ui()
 
 
+vms = []
+
+
+@pytest.fixture(scope="module")
+def vm_list(appliance, provider):
+    """Fixture to provide list of vm objects"""
+    # TODO: Need to add list of vm and its configuration in cfme_data.yaml
+    templates = [provider.data.templates.big_template['name']]
+    for template in templates:
+        vm_name = random_vm_name(context='v2v-auto')
+        collection = appliance.provider_based_collection(provider)
+        vm = collection.instantiate(vm_name, provider, template_name=template)
+
+        if not provider.mgmt.does_vm_exist(vm_name):
+            logger.info("deploying {} on provider {}".format(vm_name, provider.key))
+            vm.create_on_provider(allow_skip="default")
```

Comment:
please add `datastore='name'` to this call. Fetch the datastore name from yamls.